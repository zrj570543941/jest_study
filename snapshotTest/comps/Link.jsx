import React from 'react';

export default () => (
    <div style={{marginLeft: 10}}>这是一个组件！abcd</div>
)
