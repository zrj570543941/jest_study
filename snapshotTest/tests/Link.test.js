import React from 'react';
import Link from '../comps/Link';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer
        .create(<Link />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
