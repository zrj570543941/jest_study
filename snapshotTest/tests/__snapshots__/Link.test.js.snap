// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  style={
    Object {
      "marginLeft": 10,
    }
  }
>
  这是一个组件！abcd
</div>
`;
